### ISSUE TITLE
react-sortable not working on IE11

### ISSUE THREAD

**Issue by joujoukinder** [clauderic/react-sortable-hoc#236]
Hi, 

I need to use react-virtualized with drag and drop supported, but I can't make the demo of react-sortable work on IE11. Drag and drop is just not enabled on the demo using IE11.

Is it possible to use react-sortable on IE11 ?

Thanks for help.

Regards,

MS

**Comment by antihero898** [comment]
I've actually just looked into this issue as I needed support for IE 11 myself as well. I'm currently using the commonjs SortableContainer, so this fix may not be exact compared to ES6 and UMD.

Quickly looking in commonjs/SortableContainer.index.js, both updatePosition and animateNodes rely on `window.scrollY`, `window.scrollX`, `this.initialWindowScroll.top`, and `this.initialWindowScroll.left` to calculate adjustment for window scroll. Unfortunately, all of these values appear to be NaN when using IE 11. Doing some quick research, a good alternative seems to be `window.pageYOffset` and `window.pageXOffset`. 

As a temporary fix, we should use the following in commonjs/SortableContainer.index.js:

**updatePosition**

```
        translate.y -= (window.scrollY - this.initialWindowScroll.top) || window.pageYOffset;
        translate.x -= (window.scrollX - this.initialWindowScroll.left) || window.pageXOffset;
```

**animateNodes**

```
        var scrollDifference = {
          top: (window.scrollY - this.initialWindowScroll.top) || window.pageYOffset,
          left: (window.scrollX - this.initialWindowScroll.left) || window.pageXOffset
        };
```

Testing this with scroll appears to work and still feels extremely smooth.

**Comment by TigerWolf** [comment]
I am also having this problem. It appears that 0.6.5 has broken IE 11. I have rolled back to 0.6.3

**Comment by Ghostavio** [comment]
confirming that `0.6.5` broke IE11, it also works fine on `0.6.4`

**Comment by humiston** [comment]
#248 appears to have fixed these issues.

**Comment by mccambridge** [comment]
@clauderic Have you had a chance to look at this. Not working on IE11 for me, either. Thanks!

**Comment by humiston** [comment]
@mccambridge I have a possible bug fix in pr #248 that @clauderic is in the process of reviewing. Unfortunately, I'm not sure when he will be free again to review my latest changes. Hopefully it is soon.

**Comment by clauderic** [comment]
This should be fixed in `0.6.6`, thanks @humiston 😄
